src/videojs-hls.js

```diff
@@ -895,6 +895,29 @@ videojs.HlsHandler.prototype.playlistUriToUrl = function(segmentRelativeUrl) {
   return playListUrl;
 };
 
+/*  Turns segment byterange into a string suitable for use in
+ *  HTTP Range requests
+ */
+videojs.HlsHandler.prototype.byterangeStr = function(byterange) {
```

Comment:
If you like this function on the HlsHandler prototype, I think it should be marked "private". We normally indicate a function is internal-only by appending an underscore.